instrumentation/opentelemetry-instrumentation-botocore/tests/cassettes/test_converse_with_content.yaml

```diff
@@ -0,0 +1,93 @@
+interactions:
+- request:
+    body: |-
+      {
+        "messages": [
+          {
+            "role": "user",
+            "content": [
+              {
+                "text": "Say this is a test"
+              }
+            ]
+          }
+        ],
+        "inferenceConfig": {
+          "maxTokens": 10,
+          "temperature": 0.8,
+          "topP": 1,
+          "stopSequences": [
+            "|"
+          ]
+        }
+      }
+    headers:
+      Content-Length:
+      - '170'
+      Content-Type:
+      - !!binary |
```

Comment:
hmm weird... is this due to http/2 or something?

Reply:
I don't think so, botocore should use urllib3 as http library and I've recorded with an old-ish `urllib3==1.26.19`. urllib3 does not have http/2 support yet